File: app/src/main/java/com/adenilson/testeandroid/networking/webservices/contact/ContactAPI.java
package com.adenilson.testeandroid.networking.webservices.contact;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Headers;

/**
 * @author: Adenilson N. da Silva Junior
 * On date: 09/06/2018
 */

public interface ContactAPI {

    @Headers("Accept: application/json")
    @GET("cells.json")
    Call<CellsResponse> getCells();

}

File: app/src/main/java/santander/com/br/invest/api/ContactApi.kt
package santander.com.br.invest.api

import io.reactivex.Observable
import retrofit2.http.GET
import santander.com.br.invest.model.CellsResponse

interface ContactApi {

  @GET("cells.json")
  fun getCells(
  ): Observable<CellsResponse>
}
